docs/parameters.md

```diff
@@ -40,5 +40,9 @@ Configuration options for the server are defined only via command-line options a
 | `log_level` | `"DEBUG"/"INFO"/"ERROR"` | Serving logging level |
 | `log_path` | `string` | Optional path to the log file. |
 | `cache_dir` | `string` | Path to the model cache storage. Caching will be enabled if this parameter is defined or the default path /opt/cache exists |
+| `config_path` | `string` |  Absolute path to json configuration file |
```

Comment:
Hmm, I wonder if this should be among `Server configuration options`. 
I'm thinking about putting it in at the bottom of `Model configuration options` section as a separate, single row table, add link to a docs describing serving multiple models and note that specifying `config_path` is mutually exclusive with putting model parameters in the CLI. But it's not a strong preference, so I'm also okay with current version. 
What do you think?

Reply:
I agree with Milosz on this. This is in multi model section in ```--help``` and in ModelSettings in C-API.